npm install geom-center-and-normalize breaks npm audit

```bash
$ npm audit
npm ERR! code ENOAUDIT
npm ERR! audit Your configured registry (https://registry.npmjs.org/) does not support audit requests, or the audit endpoint is temporarily unavailable.
```
Weird one but I guessed caused by:
```jsonc
"pex-geom": "git://github.com/variablestudio/pex-geom",
"pex-math": "git://github.com/variablestudio/pex-math"
```
on the wrong organisation.

--- a/index.js
+++ b/index.js
@@ -1,17 +1,17 @@
-var AABB = require('pex-geom/AABB')
-var Vec3 = require('pex-math/Vec3')
+var aabb = require('pex-geom/aabb')
+var vec3 = require('pex-math/vec3')
 
 function centerAndNormalize (positions) {
-  var aabb = AABB.fromPoints(positions)
-  var center = AABB.center(aabb)
-  var size = AABB.size(aabb)
+  var aabb = aabb.fromPoints(positions)
+  var center = aabb.center(aabb)
+  var size = aabb.size(aabb)
   var scale = Math.max(size[0], Math.max(size[1], size[2]))
 
   var newPositions = []
   for (var i = 0; i < positions.length; i++) {
-    var p = Vec3.copy(positions[i])
-    Vec3.sub(p, center)
-    Vec3.scale(p, 1 / scale)
+    var p = vec3.copy(positions[i])
+    vec3.sub(p, center)
+    vec3.scale(p, 1 / scale)
     newPositions.push(p)
   }
   return newPositions

--- a/index.js
+++ b/index.js
@@ -2,9 +2,9 @@ var aabb = require('pex-geom/aabb')
 var vec3 = require('pex-math/vec3')
 
 function centerAndNormalize (positions) {
-  var aabb = aabb.fromPoints(positions)
-  var center = aabb.center(aabb)
-  var size = aabb.size(aabb)
+  var result = aabb.fromPoints(positions)
+  var center = aabb.center(result)
+  var size = aabb.size(result)
   var scale = Math.max(size[0], Math.max(size[1], size[2]))
 
   var newPositions = []

--- a/package.json
+++ b/package.json
@@ -11,8 +11,8 @@
   },
   "dependencies": {
     "clone": "^1.0.2",
-    "pex-geom": "git://github.com/variablestudio/pex-geom",
-    "pex-math": "git://github.com/variablestudio/pex-math"
+    "pex-geom": "^2.0.2",
+    "pex-math": "^2.1.1"
   },
   "devDependencies": {},
   "scripts": {

